ansible_collections/arista/avd/docs/porting-guides/5.x.x.md

```diff
@@ -289,6 +289,43 @@ By default an MLAG ID will only be configured on Port-Channel downlinks dual-hom
 + mlag_on_orphan_port_channel_downlink: true
 ```
 
+### Changes in MTU values and support for `p2p_uplinks_mtu` in platform settings
+
+#### Default MTU value changed from `9214` to `9194` for Dps1 and WAN HA interfaces
+
+In AVD 4.0.0, the default MTU for Dps1, LAN Uplink interfaces, and WAN HA interfaces was `9214`.
+In AVD 5.0.0, the default MTU for Dps1 and WAN HA interfaces has been changed to `9194`.
+
+#### New Feature: `p2p_uplinks_mtu` support in platform settings
+
+In AVD 5.0.0, support for `p2p_uplinks_mtu` has been added in platform settings. The precedence for setting MTU on point-to-point uplink interfaces is now as follows:
+
+1. `node_type_uplink_mtu`
+2. `platform_p2p_uplinks_mtu`
+3. `p2p_uplinks_mtu`
+4. Default value: `9214`
+
+MTU on WAN HA interfaces
+
+```diff
+wan_router:
+  node_groups:
+    - group: SITE_HA_ENABLED
+      wan_ha:
+        enabled: true
+        ha_interfaces: [Ethernet52]
++       mtu: 9214
+```
+
+MTU on point to point uplink interfaces via platform settings
+
+```diff
+p2p_uplinks_mtu: 9194
+platform_settings:
+  - platforms: [ default ]
++   p2p_uplinks_mtu: 9214
+```
```

Comment:
so these are examples to retain previous (faulty) existing behavior right? maybe they should belong in the previous section?

Reply:
I think we need to combine these two sections into one, only concerning the change and how to revert it. It is fine with the details on precedence, since it can affect the workaround as well.

Reply:
updated